CHANGELOG.md

```diff
@@ -2,10 +2,16 @@
 
 ## unreleased
 
+* PayPalButtons
+  * Add `PayPalCreditButtonColor.BLUE` enum case
 * Breaking Changes
   * PayPalButtons
     * Remove `PaymentButtonSize` enum
     * Remove `size` property from `PaymentButton` base class
+    * Remove `PaymentButtonColor.GOLD` enum case
+    * Remove `PaymentButtonColor.SILVER` enum case
+    * Remove `PayPalCreditButtonColor.GOLD` enum case
+    * Remove `PayPalCreditButtonColor.DARK_BLUE` enum case
```

Comment:
Adding these items to the CHANGELOG. They'll probably need to be updated as we do more refactoring.